### PR TITLE
Split OS X clients on the downloads page by Java version

### PR DESCRIPTION
**Pull request by sbesson**
Following upstream work in https://github.com/openmicroscopy/openmicroscopy/pull/3407 and https://github.com/openmicroscopy/openmicroscopy/pull/3337 two sets of OS X clients are now produced: for Java 6 and Java 7+.

This PR reflects this change by adding an extra row in the clients table of the downloads page.

### CODE REVIEW

**Comment by sbesson** [comment]
Note:@gusferguson is working on new icons to reflect the different OS X clients.

File: omero_downloads.html [hunk 1]
```diff
@@ -151,9 +161,16 @@
                                 >OMERO.importer</a> and <a
                                 href="@DOC_URL@/users/clients-overview.html#omero-editor"
                                 >OMERO.editor</a> and requires Java Version
-                                <strong>1.6</strong> or higher. OMERO.web is
-                            part of the server package, so individual users do
-                            not need to install it locally.</p>
+                                <strong>1.6</strong> or higher.</p>
+                    </li>
+                    <li>
+                        <p> Mac OS X users should use the clients matching
```
Comment (ghost) [inline omero_downloads.html]: Maybe also add a sentence here with a link to http://download.oracle.com/otn-pub/java/jdk/8u31-b13/jre-8u31-macosx-x64.dmg or http://www.oracle.com/technetwork/java/javase/downloads/jre8-downloads-2133155.html e.g. "MacOS X 10.7 and later should install the latest Java and use the Java7+ downloads.  MacOS X 10.6 users can only use the Java6 download".  This just makes it clearer which versions work with which MacOS X version, and how to get it without reading the docs.

**Comment by ghost** [comment]
This looks fine.  Just one minor comment about MacOS.

**Comment by joshmoore** [comment]
Minor question re: the icon: was omitting the "+" from "Java7" intentional?

**Comment by gusferguson** [comment]
Hi Josh - no - will send revised - if I can fit it in!

Regards
Gus

**Comment by sbesson** [comment]
For pushed with @gusferguson's new icons and the amended OSX text.

**Comment by hflynn** [comment]
Page for RC1 looks good. One minor point for tidiness/consistency - since we now have the second bullet point regarding the Java versions, can you change 'Java Version 1.6 or higher' in the 1st bullet point above to 'Java6 or higher'

**Comment by hflynn** [comment]
Other than that tiny nitpick, looks good!

**Comment by sbesson** [comment]
Pushed.

**Comment by pwalczysko** [comment]
@sbesson I just discussed with @rleigh-dundee  and he suggests to put a link in the downloads page
https://downloads.openmicroscopy.org/omero/5.0.7-rc2/
where it says
`Mac OS X users should use the clients matching their installed Java version. MacOS X 10.7 and later should install the latest Java and use the Java 7+ downloads. MacOS X 10.6 users can only use the Java 6 download.`
To put the link to the lates Java downloads.
@manics suggests  even better…autodetect the user's OS and highlight the appropriate client download

**Comment by hflynn** [comment]
I'd go for http://www.oracle.com/technetwork/java/javase/downloads/index.html

**Comment by ghost** [comment]
I suggest "install the latest Java" should be "install the latest Java Runtime Environment (JRE)" so that question of what to download is less confusing.  The general index page for downloads has a lot of different download links, so this makes it clearer what to get.

**Comment by hflynn** [comment]
Looks good to me!

**Comment by pwalczysko** [comment]
Looks good to me too.

**Comment by hflynn** [comment]
Good to merge once Travis is green

**Comment by sbesson** [comment]
--rebased-to #135

**Pull request merged**